bpf/bpf_xdp.c

```diff
@@ -112,24 +112,42 @@ bpf_xdp_exit(struct __ctx_buff *ctx, const int verdict)
 	return verdict;
 }
 
+#ifdef XDP_TRACING
+static __always_inline void __maybe_unused
+send_xdp_trace_notify(struct __ctx_buff *ctx __maybe_unused)
+{
+	send_trace_notify(ctx, TRACE_FROM_NETWORK, 0, 0, 0, ctx->ingress_ifindex,
```

Comment:
nit: maybe only wrap the `send_trace_notify()` block around `#ifdef` rather than redefining the function twice